settings.gradle.kts

```diff
@@ -116,8 +117,7 @@ dependencyResolutionManagement {
     // runtime.
     create("libs") {
       // The HAPI API version to use, this need to match the tag set on gitRepositories above
-      // this looks for a tag in nexus repository manager
-      version("hapi-version", "0.38.1-SNAPSHOT")
+      version("hapi-version", "0.38.0-allowance-SNAPSHOT")
```

Comment:
Is this still going backwards from `0.38.1-...` to `0.38.0-...`?

Reply:
ah yes :( Updated. I should merge this PR soon 😀